Kudu.Core/Functions/KedaFunctionTriggerProvider.cs

```diff
@@ -150,5 +178,61 @@ public static string GetKedaTriggerType(string triggerType)
                     return triggerType;
             }
         }
+        
+        private static bool IsDurable(FunctionTrigger function) => 
+            function.Type.Equals("orchestrationTrigger", StringComparison.OrdinalIgnoreCase) ||
+            function.Type.Equals("activityTrigger", StringComparison.OrdinalIgnoreCase) ||
+            function.Type.Equals("entityTrigger", StringComparison.OrdinalIgnoreCase);
+
+        private static bool TryGetDurableKedaTrigger(string hostJsonText, out ScaleTrigger scaleTrigger)
+        {
+            scaleTrigger = null;
+            if (string.IsNullOrEmpty(hostJsonText))
+            {
+                return false;
+            }
+
+            JObject hostJson = JObject.Parse(hostJsonText);
+
+            // Reference: https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-bindings#durable-functions-2-0-host-json
+            JObject storageProviderConfig = hostJson.SelectToken("extensions.durableTask.storageProvider") as JObject;
+            string storageType = storageProviderConfig?["type"]?.ToString();
+
+            // Custom storage types are supported starting in Durable Functions v2.4.2
+            if (string.Equals(storageType, "MicrosoftSQL", StringComparison.OrdinalIgnoreCase))
```

Comment:
nit: Same here. Move strings to constants.

Reply:
Done